src/com/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -297,6 +298,7 @@
 
     ;; Initialize database-dependent metrics and dlo metrics if existent.
     (pop/initialize-metrics write-db)
+    (alter-var-root (var scf-store/*gc-chunksize*) (constantly gc-chunksize))
```

Comment:
this is stupid; planning to change it. I ran into some really strange issues with call counting in command testing when I tried to thread this through as an argument. I'll pick this up later and figure out what the problem was.